pigsty.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "bundler/setup"
+require "active_support/core_ext/string/inflections"
+require "pgpm"
+require "csv"
+require "open-uri"
+
+URI.open("https://raw.githubusercontent.com/pgsty/extension/refs/heads/main/data/pigsty.csv") do |f|
+  i = 0
+  CSV.foreach(f, headers: true) do |row|
+    next unless row["url"] =~ /github/ || row["url"] =~ /gitlab/
+
+    origin = row["url"]
+    if row["url"] =~ /github/
+      github = origin.match(%r{https://github\.com/([^/]+)/([^/]+)}).captures.join("/")
+      origin = "github \"#{github}\""
```

Comment:
Idea: group by github workspace to also detect workspaces for cases like omnigres/timescale.

Reply:
🤔 Maybe we can do this on a case-by-case scenario. Like: should we do this for personal github accounts? What should we do for non-github projects?